src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V63/Messages/ReceiptsMessageSerializer.cs

```diff
@@ -43,7 +43,8 @@ public void Serialize(IByteBuffer byteBuffer, ReceiptsMessage message)
                         b.Select(
                             n => n is null
                                 ? Rlp.OfEmptySequence
-                                : _decoder.Encode(n, _specProvider.GetSpec(n.BlockNumber).IsEip658Enabled ? RlpBehaviors.Eip658Receipts : RlpBehaviors.None)).ToArray())).ToArray());
+                                // for TxReceipt there is no timestamp, as such, we are keeping the old implementation. wonder how we can metigate this later if future EIPs affecting this are added. 
```

Comment:
receipts are per block, so we could potentially know the timestamp from block/header.

Reply:
Yeah, i understand that. but the block information is not included in the receipt message. not sure what to do here honestly.

Reply:
As far as my understanding go. these messages are transmitted between nodes during sync?

Reply:
@smartprogrammer93 it is easy. You can do similar trick like for 1559